index.html

```diff
@@ -44,5 +45,152 @@ <h2>LET’S CONNECT</h2>
         </ul>
       </div>
     </section>
+
+    <!--Works section-->
+    <section id="works">
+      <ul class="works-container">
+        <li class="work-card">
+          <img src="/media/snapshoot-portfolio.png" alt="Work preview for project Tonic" class="work-card-img">
+          <h1 class="work-card-tittle">Tonic</h1>
+          <div class="work-card-info">
+            <h2>Canopy</h2>
+            <span class="circle"></span>
+            <h3>Back End Dev</h3>
+            <span class="circle"></span>
+            <h3>2015</h3>
+          </div>
+          <p class="work-card-description">
+            A daily selection of privately 
+            personalized reads; no accounts or sign-ups required.
+          </p>
+          <ul class="work-card-tags">
+            <li class="tag">html</li>
+            <li class="tag">css</li>
+            <li class="tag">javaScript</li>
+          </ul>
+          <button type="button" class="work-card-seeProject">See Project</button>
+        </li>
+
+        <li class="work-card">
+          <img src="/media/snapshoot-portfolio-1.png" alt="Work preview for project Multi-Post Stories" class="work-card-img">
+          <h1 class="work-card-tittle">Multi-Post Stories</h1>
+          <div class="work-card-info">
+            <h2>Canopy</h2>
+            <span class="circle"></span>
+            <h3>Back End Dev</h3>
+            <span class="circle"></span>
+            <h3>2015</h3>
+          </div>
+          <p class="work-card-description">
+            A daily selection of privately 
+            personalized reads; no accounts or sign-ups required.
+          </p>
+          <ul class="work-card-tags">
+            <li class="tag">html</li>
+            <li class="tag">css</li>
+            <li class="tag">javaScript</li>
+          </ul>
+          <button type="button" class="work-card-seeProject">See Project</button>
+        </li>
+
+        <li class="work-card">
+          <img src="/media/snapshoot-portfolio-2.png" alt="Work preview for project Tonic" class="work-card-img">
```

Comment:
**[ OPTIONAL ]**

- I think something is missing for each of the projects: the project screenshots are not rendering. I believe this is due to the `src` value of each of the image tags. What do you think :thinking:? Kindly fix and test to ensure that all the images are rendering as expected. Rendering the images will make your portfolio look more like the design, and in turn, will allow your users to see what the project looks like before checking it out.

Reply:
This is weird, the images are rendering correctly on my side. I just commented in other of your suggestions with a screenshot of how my button is loading with the proper width, you can see there that the image is also rendered correctly.

Reply:
:+1: The images now are rendering :heavy_check_mark: 
=> Just for confirmation check out [this link](https://rawcdn.githack.com/D4nielJ/developer-portfolio/e55fecfc73836699002a4f10591a02baa638937f/index.html) It represents the current live version of the project